demo/protocol-demo/src/demonstrator.rs

```diff
@@ -129,7 +129,7 @@ impl Party {
             "Party #{}: {}",
             self.party_id,
             if let Some(sig) = &signature {
-                format!("lottery #{:?} won", sig.indexes)
+                format!("lottery #{:?} won", sig.concatenation_signature.indexes)
```

Comment:
We could have an accessor method and avoid exposing internal representation of the single signature?